ClientV2Tests/webpack.config.js

```diff
@@ -27,7 +27,11 @@ module.exports = function (evn, argv) {
     port: 8081, // where to the run dev-server
     proxy: {
       '/api/*': { // tell webpack-dev-server to re-route all requests from client to the server
-        target: "http://localhost:8080",// assuming the suave server is hosted op port 8080
+        target: "http://localhost:8397",// assuming the suave server is hosted op port 8080
```

Comment:
I don't think we should change this because the port of the server of the integration tests is always random. Instead, when running integration tests server, if not in headless mode, then it should choose 8080 by default:
```fs
if not (Array.contains "--headless" argv) then
  printfn "Starting web server..."
  let config = { suaveConfig with bindings  = [ HttpBinding.createSimple HTTP "127.0.0.1" 8080 ]
  startWebServer config testWebApp
  0
else
  ...
```

Reply:
Yeah I figured that's what it should do, but I couldn't get the live tests to run without changing it to that address ¯\_(ツ)_/¯. I'll change it back though since I'm done testing them anyways. Speaking of which there was one test still failing the live version that didn't fail when running normally which is the auth test. It originally failed because the dev server was missing that path, but after that it was still failing.

Reply:
That test never worked for me either so I just ignore it.

Reply:
It works fine (and passes) for the non-live version, if you're referring to the live testing it didn't actually run webpack-dev-server, but I fixed the fsx